README.md

```diff
@@ -188,10 +97,7 @@ matching dependencies:
 ```bash
 $ cd $GOPATH/src/github.com/decred/dcrstakepool
 $ git pull
-$ dep ensure
-$ go build
-$ cd $GOPATH/src/github.com/decred/dcrstakepool/backend/stakepoold
-$ go build
+$ go build . ./backend/stakepool
 ```
 
 ## Setup
```

Comment:
Totally valid suggestions. I've opened #508 for these because the suggestions aren't directly related to the changes in this PR and shouldn't need to block this PR going in.